roles/cloud-bootstrap/tasks/network.yml

```diff
@@ -3,26 +3,34 @@
   os_network:
     cloud: "{{ cloud }}"
     name: control-plane
+    auth:
+      project_name: "{{ prod_project }}"
```

Comment:
Right but this will be run from an admin locally and may have a different project (their personal) set on the cloud their using.